public/javascripts/data/storyManager.js

```diff
@@ -0,0 +1,45 @@
+const Database = require('better-sqlite3');
+
+exports.displayAllStories = () => {
+    
+    // Get all stories (an array of objects)
+    let db = new Database('public/javascripts/data/stories.db');
+    let allStories = db.prepare('SELECT * FROM stories').all();
+
+    // Get the first five paragraphs (if that many) from each story.
+    // I add a 'paragraphs' property (an array) to the story objects.
+    // I then push all paragraphs from that story into the array.
+    for(let i = 1; i < allStories.length + 1; i++) {
+        let paraArry = [];
+        let paraStmt = db.prepare('SELECT * from paragraphs WHERE stories_id = (?) LIMIT 5');
+        paraArry.push(paraStmt.all(i));
+        allStories[i - 1]['paragraphs'] = []
+        for(let j = 0; j < paraArry[0].length; j++) {
+            allStories[i - 1]['paragraphs'].push([paraArry[0][j]['content']]);
+        }   
+    };
+   
+    db.close();
+    return(allStories);
+};
+
+// Function writes a new story (title) to the database
+exports.addStory = (storyTitle, userId) => {
+
+    let db = new Database('public/javascripts/data/stories.db');
+    const new_user = db.prepare('INSERT INTO stories (user_id, title) VALUES (?, ?)').run(userId, storyTitle);
+    db.close();
+    return;
```

Comment:
Do you want to return the story from the insert? Consider adding a `RETURNING` clause to your `INSERT` statement

https://www.sqlite.org/lang_returning.html

Reply:
Here after inserting the story into the database, I'm redirecting back to the home page which displays all of the stories.

`router.get('/', handleAuth.requireLogin, handleAuth.currentUser, stories.getAllStories);`

```
exports.addStory = function(req, res, next) {

    let storyTitle = req.body.addStoryInput;
    let userId = req.session.user['id'];
    
    storyManager.addStory(storyTitle, userId);
    res.redirect('/');
};
```